apps/masters/serializers.py

```diff
@@ -75,45 +116,74 @@ class Meta:
         model = Transporters
         fields = ['transporter_id', 'name', 'code', 'gst_no', 'website_url']
```

Comment:
what are these fields? why only few fields inthe list?

Reply:
It is done by pramod